CPAC/resources/configs/pipeline_config_blank.yml

```diff
@@ -752,8 +752,7 @@ registration_workflows:
       # Choose FSL or ABCD as coregistration method
       using: FSL
 
-      # Choose brain or whole-head as coregistration input
-      input: brain
+      #TODO Add input field here to choose between whole head or brain
```

Comment:
Do we want this comment in the release?

Reply:
I'm partial to just having an open issue for tracking, but I get the point of having a _TODO_ comment in there.

Since the pipeline configs are something users interact directly, I lean away from having it in the release (I think it could be confusing), but it's not a strong preference.

Reply:
> Thanks!
> 
> I opened #2203 to replace this `TODO`

― https://github.com/FCP-INDI/C-PAC/pull/2201#pullrequestreview-2726003496